src/OSPSuite.Core/Domain/Services/PKValuesCalculator.cs

```diff
@@ -153,7 +153,9 @@ private void addDynamicPKValues(ICache<string, double> pk, IReadOnlyList<Dynamic
       {
          foreach (var dynamicPKParameter in dynamicPKParameters)
          {
-            var startTime = dynamicPKParameter.EstimateStartTimeFrom(options);
+            //No start time specified? Then we assume we want to calculate from the beginning of the time array
+            var startTime = dynamicPKParameter.EstimateStartTimeFrom(options) ?? time.First();
```

Comment:
I think this makes sense don't you think? If no time is specified, we assume beginning of time arrray?

Reply:
yep